addon/components/frost-select-dropdown.js

```diff
@@ -125,6 +126,26 @@ export default Component.extend(PropTypeMixin, {
     }
   },
 
+  _startUpdating () {
```

Comment:
I'd like to see this written using `ember-concurrency` if possible - more safety mechanisms built in and that's how I would like to see all tasks written